src/System.Linq.Expressions/tests/TestExtensions/TestOrderer.cs

```diff
@@ -16,7 +16,7 @@ IEnumerable<TTestCase> ITestCaseOrderer.OrderTestCases<TTestCase>(IEnumerable<TT
             Dictionary<int, List<TTestCase>> queue = new Dictionary<int, List<TTestCase>>();
             foreach (TTestCase testCase in testCases)
             {
-                Xunit.Abstractions.IAttributeInfo orderAttribute = ReflectionAbstractionExtensions.GetCustomAttributes(testCase.TestMethod.Method, typeof(TestOrderAttribute)).FirstOrDefault();
+                Xunit.Abstractions.IAttributeInfo orderAttribute = testCase.TestMethod.Method.GetCustomAttributes(typeof(TestOrderAttribute)).FirstOrDefault();
```

Comment:
After I changes some of default references this failed to compile and I looked at the history of the file and for some reason when you ported this originally you changed it from the extension method call to a static call so I changed it back here. I also decided to directly add the necessary reference to the xunit.execution directly to this project instead of as part of the default references.